src/commonMain/kotlin/teksturepako/pakku/cli/cmd/Set.kt

```diff
@@ -15,25 +15,31 @@ import teksturepako.pakku.api.projects.UpdateStrategy
 
 class Set : CliktCommand()
 {
+    override val printHelpOnEmptyArgs = true
+
     override fun help(context: Context) = "Set various properties of your modpack or projects"
 
     // -- PROJECTS --
 
-    private val projectArgs: List<String> by argument("projects").multiple()
+    private val projectArgs: List<String> by argument(
+        "projects",
+        help = "Use the config file (pakku.json) or 'cfg' command instead."
```

Comment:
This should also use the `.deprecated()` extension function

Reply:
There isn't `.deprecated()` for the argument xd

Reply:
Oh, okay never mind then. It's weird that there isn't.